scripts/prepare-machine.ps1

```diff
@@ -555,6 +555,8 @@ if ($IsLinux) {
 }
 
 if ($IsMacOS) {
+    brew update
+    brew install cmake
```

Comment:
We don't need this for automation, so I'm hesitant to have this always run. Any downsides?

Reply:
Brew isn’t something built into the os, and it’s possible to grab cmake without it. So it should probably only run if brew is actually installed.

Reply:
@nibanks 
I'm trying to use curl here, but how does automation download/install cmake?
I want to know which specific cmake version the automation use. If it is using latest, I want the URL because there is no latest URL as much as I checked.

Reply:
We use PowerShell to download stuff.

Reply:
Removed the change as this is mentioned in BUILD.md